packages/edit-site/src/components/sidebar-edit-mode/page-panels/index.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	PanelBody,
+	__experimentalVStack as VStack,
+	Button,
+} from '@wordpress/components';
+import { page as pageIcon } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+import { humanTimeDiff } from '@wordpress/date';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEntityRecord } from '@wordpress/core-data';
+import { BlockContextProvider, BlockPreview } from '@wordpress/block-editor';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../../store';
+import useEditedEntityRecord from '../../use-edited-entity-record';
+import removePageFromBlockContext from '../../../utils/remove-page-from-block-context';
+import SidebarCard from '../sidebar-card';
+import ContentBlocksList from './content-blocks-list';
+
+export default function PagePanels() {
+	const context = useSelect(
+		( select ) => select( editSiteStore ).getEditedPostContext(),
+		[]
+	);
+
+	const { hasResolved: hasPageResolved, editedRecord: page } =
+		useEntityRecord( 'postType', context.postType, context.postId );
+
+	const {
+		isLoaded: isTemplateLoaded,
+		getTitle: getTemplateTitle,
+		record: template,
+	} = useEditedEntityRecord();
+
+	const { togglePageContentLock } = useDispatch( editSiteStore );
+
+	const blockContext = useMemo(
+		() => removePageFromBlockContext( context ),
+		[ context ]
+	);
+
+	if ( ! hasPageResolved || ! isTemplateLoaded ) {
+		return null;
+	}
+
+	return (
+		<>
+			<PanelBody>
+				<SidebarCard
+					title={ page.title }
+					icon={ pageIcon }
+					description={ sprintf(
+						// translators: %s: Human-readable time difference, e.g. "2 days ago".
+						__( 'Last edited %s' ),
+						humanTimeDiff( page.modified )
+					) }
+				/>
+			</PanelBody>
+			<PanelBody title={ __( 'Content' ) }>
+				<ContentBlocksList />
+			</PanelBody>
+			<PanelBody
+				className="edit-site-edit-template-panel"
+				title={ __( 'Template' ) }
+			>
+				<VStack>
+					<div>{ getTemplateTitle() }</div>
+					<div className="edit-site-edit-template-panel__preview">
+						<BlockContextProvider value={ blockContext }>
+							<BlockPreview
+								viewportWidth={ 1024 }
+								blocks={ template.blocks }
+							/>
+						</BlockContextProvider>
+					</div>
+					<Button
+						className="edit-site-edit-template-panel__button"
+						variant="secondary"
+						onClick={ () => togglePageContentLock() }
```

Comment:
```suggestion
						onClick={ () => togglePageContentLock( false ) }
```

We can be explicit here about the [toggle value](https://github.com/WordPress/gutenberg/blob/599a4d1ab24d295371232cb992cc274043099612/packages/edit-site/src/store/actions.js#L542-L542) if we're just toggling things off.